### PR TITLE
Add Rc-Template

### LINKED ISSUE #3: Creation of Template Circuits
As a new CEG/EE Student, I can **create template circuits** using common electrical components such as Resistors, Capacitors, Inductors so that I can analyse common configurations.

Based on the [User Guide section](https://github.com/AY2021S1-CS2113T-W13-3/tp/blob/master/docs/UserGuide.md#create-circuit-template-template):

> Creates a circuit template.
> 
> Format: `template TEMPLATE`
> 
> * The `TEMPLATE` can be chosen from the following 4 - `r`, `rc`, `rl`, `lc`
>     * `r` only consists of the resistor.
>     * `rc` consists of the resistor and capacitor.
>     * `rl` consists of the resistor and inductor.
>     * `lc` consists of the inductor and capacitor.
> 
> Example of usage:
> 
> `template rc`
> 
> Expected outcome:
> 
> ```
> +---R-----C---+
> |             |
> |             |
> +----+V_ac+---+
> ```

### PR DESCRIPTION
**Pull request by PraveenElango**
Include template that provides calculations for RC-circuit, i.e. circuit with resistor and capacitor

### CODE REVIEW

File: src/main/java/seedu/duke/template/RcTemplate.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        capacitor = new Capacitor(capacitance);
+        resistor = new Resistor(resistance);
+    }
+
+    /**
+     * Returns capacitor object, an attribute of the instance of RcTemplate.
+     *
+     * @return capacitor, an instance of the Capacitor class.
+     */
+
```
Comment (hughjazzman) [inline src/main/java/seedu/duke/template/RcTemplate.java]: Consider removing the line between the JavaDoc and method? From the coding standard: https://se-education.org/guides/conventions/java/basic.html#comments
> No blank line between the documentation block and the method/class
```suggestion
```

File: src/main/java/seedu/duke/template/RcTemplate.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    public Resistor getResistor() {
+        return resistor;
+    }
+
+    /**
+     * Returns String consisting of total capacitance and resistance values of the RcTemplate object.
+     *
+     * @return String representation of RcTemplate object.
+     */
+
```
Comment (hughjazzman) [inline src/main/java/seedu/duke/template/RcTemplate.java]:
```suggestion
```

File: src/main/java/seedu/duke/template/RcTemplate.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    public Capacitor getCapacitor() {
+        return capacitor;
+    }
+
+    /**
+     * Returns resistor object, an attribute of the instance of LrTemplate.
+     *
+     * @return resistor, an instance of the Resistor class.
+     */
+
```
Comment (hughjazzman) [inline src/main/java/seedu/duke/template/RcTemplate.java]:
```suggestion
```

File: src/main/java/seedu/duke/template/RcTemplate.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+public class RcTemplate extends Template {
+    private static final String RC_TEMPLATE = "\t+---R-----C---+\n"
+            + "\t|             |\n"
+            + "\t|             |\n"
+            + "\t+----+V_ac+---+\n";
+
+    private Resistor resistor;
+    private Capacitor capacitor;
```
Comment (hughjazzman) [inline src/main/java/seedu/duke/template/RcTemplate.java:14]: Could consider having setters for these? @sevenseasofbri `LrTemplate` could use them as well?

**Review by hughjazzman — COMMENTED** [review]
Just small changes.

File: src/main/java/seedu/duke/template/RcTemplate.java (hunk 4 above)
Comment (sevenseasofbri) [reply src/main/java/seedu/duke/template/RcTemplate.java:14]: Yup was thinking of setters, wasn't sure whether I should add. Will add after tdys meeting!

File: src/main/java/seedu/duke/template/RcTemplate.java (hunk 1 above)
Comment (PraveenElango) [reply src/main/java/seedu/duke/template/RcTemplate.java]: Got it, thanks for pointing out Wira!

**Pull request merged**